src/commands/createFunction/durableSteps/DurableProjectConfigureStep.ts

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra, AzureWizardExecuteStep, callWithTelemetryAndErrorHandling, IParsedError, parseError } from '@microsoft/vscode-azext-utils';
+import * as path from "path";
+import { Progress, window } from 'vscode';
+import { ConnectionKey, DurableBackend, hostFileName, ProjectLanguage } from '../../../constants';
+import { viewOutput } from '../../../constants-nls';
+import { ext } from '../../../extensionVariables';
+import { IHostJsonV2 } from '../../../funcConfig/host';
+import { MismatchBehavior, setLocalAppSetting } from '../../../funcConfig/local.settings';
+import { localize } from '../../../localize';
+import { cpUtils } from '../../../utils/cpUtils';
+import { durableUtils, netheriteUtils, sqlUtils } from '../../../utils/durableUtils';
+import { pythonUtils } from '../../../utils/pythonUtils';
+import { venvUtils } from '../../../utils/venvUtils';
+import { IFunctionWizardContext } from '../IFunctionWizardContext';
+
+export class DurableProjectConfigureStep<T extends IFunctionWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 225;
+
+    public async execute(context: T, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
+        const configuring: string = localize('configuringDurableProject', 'Configuring durable project settings...');
+        progress.report({ message: configuring });
+
+        await this._configureHostAndLocalSettingsJson(context);
+        await this._tryInstallDurableDependencies(context);
+    }
+
+    public shouldExecute(context: T): boolean {
+        return !!context.newDurableStorageType;
+    }
+
+    private async _configureHostAndLocalSettingsJson(context: T): Promise<void> {
+        const hostJsonPath: string = path.join(context.projectPath, hostFileName);
+
+        if (!await AzExtFsExtra.pathExists(hostJsonPath)) {
+            const message: string = localize('durableHostConfigFailed', 'Unable to find and configure "{0}" in your project root. You may need to configure your durable function settings manually.', hostFileName);
+            ext.outputChannel.appendLog(message);
+
+            await callWithTelemetryAndErrorHandling('durableHostConfigFailed', async () => {
+                const notification: string = localize('failedToConfigureHost', 'Failed to configure your "{0}"', hostFileName);
+                void window.showInformationMessage(notification, viewOutput).then(result => {
```

Comment:
Not 100% sure that it's worth showing a notification for this, but if we do, we should use a warning message since it will cause the project to fail.

Reply:
Do we want a more silent failure where we only post the issue to the `ext.outputChannel`?

Reply:
I don't know, generally we try to avoid notifications when possible, but I agree that this is a breaking problem so we should be more obvious about it.

If we ever move it into the activity log, we can remove the notification then since that is more obvious than the output channel.